packages/amplify_auth_cognito/example/lib/main.dart

```diff
@@ -119,14 +125,20 @@ class _MyAppState extends State<MyApp> {
       print(
           'Amplify was already configured. Looks like app restarted on android.');
     }
+
+    try {
+      await Amplify.addPlugin(auth);
+    } catch (e) {
```

Comment:
Why is this happening after the `configure`?

Reply:
Forgot to remove this after testing... will remove.